lib/models/__tests__/studio.test.js

```diff
@@ -0,0 +1,37 @@
+const Studio = require('../studio');
+
+describe('is a studio', () => {
+  it('valid model', () => {
+    const data = {
+      name: 'MGM',
+      address: {
+        city: 'beaverton',
+        state: 'OR',
+        country: 'USA'
+      }
+    };
+
+    const studio = new Studio(data);
+
+    const errors = studio.validateSync();
+    expect(errors).toBeUndefined();
+
+    const json = studio.toJSON();
+    expect(json).toEqual({
+      ...data,
+      _id: expect.any(Object),
+    });
+  });
+
+  it('invalid model', () => {
+    const data = {
+      name: {},
+      address: []
+    };
+    const studio = new Studio(data);
+
+    const errors = studio.validateSync();
+    expect(errors).toBeTruthy();
```

Comment:
it is valuable to explicitly call out which errors you'll get here so that you can make sure everything that's required is throwing an error.